todo_list/admin.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+
+from todo_list.models import Task, Tag
+
+# Register your models here.

```

Comment:
Delete this comment, there is no need for it.

Reply:
Thanks!